openedx_caliper_tracking/tasks.py

```diff
@@ -29,6 +31,27 @@
 
 MAXIMUM_RETRIES = getattr(settings, 'CALIPER_KAFKA_SETTINGS', {}).get('MAXIMUM_RETRIES', 3)
 
+# SERVICE_VARIANT specifies name of the variant used, which decides what JSON
+# configuration files are read during startup.
+SERVICE_VARIANT = os.environ.get('SERVICE_VARIANT', None)
+
+# CONFIG_ROOT specifies the directory where the JSON configuration
+# files are expected to be found. If not specified, use the project
+# directory.
+CONFIG_ROOT = path(os.environ.get('CONFIG_ROOT'))
+
+# CONFIG_PREFIX specifies the prefix of the JSON configuration files,
+# based on the service variant. If no variant is use, don't use a
+# prefix.
+CONFIG_PREFIX = SERVICE_VARIANT + "." if SERVICE_VARIANT else ""
+
+########################## NON-SECURE ENV CONFIG ##############################
+# Things like server locations, ports, etc.
+
+with open(CONFIG_ROOT / CONFIG_PREFIX + "env.json") as env_file:
+    ENV_TOKENS = json.load(env_file)
+
+LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
```

Comment:
No need to do all of this yourself. You can access anything from edx-platform settings using
```
from django.conf import settings

settings.LMS_ROOT_URL
```